java/client/src/main/java/glide/api/commands/servermodules/GlideJson.java

```diff
@@ -0,0 +1,411 @@
+     * @param paths List of paths within the JSON document.
+     * @return Returns a stringifies JSON object in bytes, in which each path is a key, and it's
+     *     corresponding value, is the value as if the path was executed in the command as a single
+     *     path. If <code>key</code> doesn't exist, returns null.
+     * @example
+     *     <pre>{@code
+     * String value = client.Json.set(client, "doc", new String[] {"$.a", "$.b"}).get();
+     * assert value.equals("{\"$.a\": [1.0], \"$.b\": [2]}");
+     * }</pre>
+     */
+    public static CompletableFuture<Object> get(BaseClient client, String key, String[] paths) {
+        return executeCommand(
+                client, ArrayTransformUtils.concatenateArrays(new String[] {JSON_GET, key}, paths));
+    }
+
+    /**
+     * Retrieves the JSON value at the specified <code>paths</code> stored at <code>key</code>.
+     *
+     * @param client The Valkey GLIDE client to execute the command.
+     * @param key The <code>key</code> of the JSON document.
+     * @param paths List of paths within the JSON document.
+     * @return Returns a stringifies JSON object in bytes, in which each path is a key, and it's
+     *     corresponding value, is the value as if the path was executed in the command as a single
+     *     path. If <code>key</code> doesn't exist, returns null.
+     * @example
+     *     <pre>{@code
+     * GlideString value = client.Json.set(client, gs("doc"), new GlideString[] {gs("$.a"), gs("$.b")}).get();
+     * assert value.equals(gs("{\"$.a\": [1.0], \"$.b\": [2]}"));
+     * }</pre>
+     */
+    public static CompletableFuture<Object> get(
+            BaseClient client, GlideString key, GlideString[] paths) {
+        return executeCommand(
+                client,
+                ArrayTransformUtils.concatenateArrays(new GlideString[] {gs(JSON_GET), key}, paths));
+    }
+
+    /**
+     * Retrieves the JSON value at the specified <code>path</code> stored at <code>key</code>.
+     *
+     * @param client The Valkey GLIDE client to execute the command.
+     * @param key The <code>key</code> of the JSON document.
+     * @param path The path within the JSON document.
+     * @param options Options for formatting the byte representation of the JSON data. See <code>
+     *     JsonGetOptions</code>.
+     * @return For JSONPath (path starts with <code>$</code>): Returns a stringifies JSON list of
+     *     bytes replies for every possible path, or a byte string representation of an empty array,
+     *     if path doesn't exist. For legacy path (path doesn't start with <code>$</code>): Returns a
+     *     byte string representation of the value in <code>path</code>. If <code>path</code> doesn't
+     *     exist, an error is raised. If <code>key</code> doesn't exist, returns null.
+     * @example
+     *     <pre>{@code
+     * String value = client.Json.set(client, "doc", "$",
+     *                  JsonGetOptions.builder()
```

Comment:
I think it would look prettier if you save the options to a variable in the example